quarkus/bom/pom.xml

```
@@ -28,7 +28,7 @@
       <dependency>
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-devtools-testing</artifactId>
-        <version>${version.io.quarkus}</version>
+        <version>${version.io.quarkus.quarkus-test-maven}</version>
```

```suggestion
        <version>${version.io.quarkus.quarkus-devtools-testing}</version>
```

If we are to follow our own guidelines we should introduce a new property specific to this artifactId like the above suggestion. The above property you've proposed is used for io.quarkus:quarkus-test-maven for example.

---

and what about renaming already existing `version.io.quarkus.quarkus-test-maven` to something more generic like `version.io.quarkus.quarkus-test`? :thinking:

---

well, we can probably share the property value, but please keep it in the semantic of version properties we get to use -> version.groupid.artifactId to have clear usage